tests/stdlib/tparseutils.nim

```diff
@@ -55,5 +55,42 @@ proc test() =
     doAssert res == @[(17, "9.123456789012344"), (18, "11.123456789012344"),
                       (17, "9.123456789012344"), (17, "8.123456789012344"),
                       (16, "9.12345678901234"), (17, "9.123456789012344")]
+
 test()
 static: test()
+
+block:  # With this included, static: test() crashes the compiler (from a
+        # VM problem with parseSize calling parseFloat).
+  template ass(x) = doAssert(x) # Avoid hint[LineTooLong]
```

Comment:
But instead of using the `ass` try not using `;` and doing 1 assert per line,
I think the `LineTooLong` hint is disabled by default anyways.